### PR TITLE
Add missing fields in `ParseKeyFromURL`.

### LINKED ISSUE #15: `ParseKeyFromURL` Is missing some of the helper functions for fields.
ParseKeyFromURL Is missing some of the helper functions for fields, refs below:
- https://github.com/google/google-authenticator/wiki/Key-Uri-Format#counter
- https://github.com/google/google-authenticator/wiki/Key-Uri-Format#algorithm
- https://github.com/google/google-authenticator/wiki/Key-Uri-Format#digits

Would be nice to have them all and use `Key` object as a building block for `TOTPConfig`/`HOTPConfig`

### PR DESCRIPTION
**Pull request by s3rj1k**
Closes: https://github.com/cristalhq/otp/issues/15

### CODE REVIEW

File: otp.go [hunk 1]
```diff
@@ -127,6 +127,46 @@ func (k *Key) Period() uint64 {
 	return 30 // If no period is defined 30 seconds is the default per (RFC 6238)
 }
 
+// Digits returns the digits value.
+func (k *Key) Digits() uint {
+	var u uint = 6 // Default value for referenced Google Authenticator implementation.
```
Comment (cristaloleg) [inline otp.go]: I think we should return `0` as a default value. Google Authenticator is not the only app for OTP codes. Also, RFC 6238 doesn't mention defaults for digit param.

File: otp.go [hunk 2]
```diff
@@ -127,6 +127,46 @@ func (k *Key) Period() uint64 {
...
+		case "SHA1":
+			return AlgorithmSHA1
+		case "SHA256":
+			return AlgorithmSHA256
+		case "SHA512":
+			return AlgorithmSHA512
+		}
+	}
+
+	return AlgorithmSHA1 // Default value for referenced Google Authenticator implementation.
```
Comment (cristaloleg) [inline otp.go]: Same here, `AlgorithmUnknown` will be more suitable.

**Review by cristaloleg — CHANGES_REQUESTED** [review]

File: otp.go (hunk 1 above)
Comment (s3rj1k) [reply otp.go]: done

File: otp.go (hunk 2 above)
Comment (s3rj1k) [reply otp.go]: done

**Review by cristaloleg — APPROVED** [review]

**Comment by cristaloleg** [comment]
Thanks!

**Pull request merged**